docs/content/en/docs/concepts/packages/_index.md

```diff
@@ -8,7 +8,7 @@ description: >
 ---
 
 {{% alert title="Note" color="primary" %}}
-*The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. To request a free trial, talk to your Amazon representative or connect with one [here](https://aws.amazon.com/contact-us/sales-support-eks/).*
```

Comment:
This change is fine, but can you rebase? I made a change to this file in https://github.com/aws/eks-anywhere/pull/3024.

Reply:
Done.